cpp/models/abm/person.h

```diff
@@ -385,24 +386,25 @@ class Person
     ScalarType get_mask_protective_factor(const Parameters& params) const;
 
     /**
-     * @brief For every #LocationType a Person has a compliance value between -1 and 1.
-     * -1 means that the Person never complies to any Mask duty at the given #LocationType.
-     * 1 means that the Person always wears a Mask a the #LocationType even if it is not required.
-     * @param[in] preferences The vector of Mask compliance values for all #LocationType%s.
+     * @brief For every #InterventionType a Person has a compliance value between -1 and 1.
+     * -1 means that the Person never complies to the Intervention.
+     * 1 means that the Person always complies to the Intervention even if it is not required.
```

Comment:
```suggestion
     * -1 means that the Person never complies with the intervention.
     * 1 means that the Person always complies with the intervention, even if it is not required.
```
Does that mean 0 implies a person complies if and only if the intervention *is* required?

Reply:
Yes, I forgot this comment. Since we don't consider voluntary compliance for the current implementation, the scale (-1 to 0) is no longer applied.